nixops/resources/__init__.py

```diff
@@ -189,6 +189,17 @@ def destroy(self, wipe=False):
         )
         return False
 
+    def clear_state(self):
+        """clear this resource state, if possible."""
+        if not self.depl.logger.confirm("are you sure you want to clear resources state `{0}`".format(self.name)):
```

Comment:
I'd prefer if we emit a more clear warning of what this does, something like:
```suggestion
        if not self.depl.logger.confirm("are you sure you want to clear resources state {}: this will only remove the resource from the local NixOPS state and the resource may still exist outside of the NixOPS database.".format(self.name)):
```